lib/state/chaintime.ts

```diff
@@ -0,0 +1,26 @@
+import { isRpcSdk } from "@zeitgeistpm/sdk-next";
+import { ChainTime } from "@zeitgeistpm/utility/dist/time";
+import { atom, getDefaultStore, useAtom } from "jotai";
+import { sdkAtom } from "lib/hooks/useSdkv2";
+import { Observable, Subscription } from "rxjs";
+
+export const chainTimeAtom = atom<ChainTime | false>(false);
```

Comment:
Why initialise as `false` instead `null` or `undefined`?

Reply:
Its a limitation of jotai/typescript. Null doesnt work somehow. Its in the jotai docs.